asset/css/markbind.css

```diff
@@ -2,17 +2,24 @@ blockquote {
     font-size: 14px;
 }
 
-.radio-list-item label {
-    font-weight: inherit;
+.radio-list-item,
+.task-list-item {
+    list-style-type: none;
+    margin: 0 0 0 -1.2em;
 }
 
 @media (max-width: 767px) {
     .navbar-default .navbar-nav .open .dropdown-menu > li > a,
     .navbar-inverse .navbar-nav .open .dropdown-menu > li > a {
         color: #333;
     }
-    
+
     .navbar-nav .open .dropdown-menu {
         background-color: #fff;
     }
 }
+
+.radio-list-input,
+.task-list-item-checkbox {
+    vertical-align: text-bottom;
+}
```

Comment:
As [mentioned](https://github.com/MarkBind/markbind-cli/pull/35#discussion_r173624526), this misaligns the input on Safari. Let's do without this.